example/lib/deviceid_sessionid.dart

```diff
@@ -16,29 +16,29 @@ class _DeviceIdSessionIdState extends State<DeviceIdSessionId> {
             Text('Device Id', style: Theme.of(context).textTheme.headlineSmall),
           ],
         ),
-        Row(
-          children: [
-            FutureBuilder(
-              future: AppState.of(context).analytics.getDeviceId(),
-              builder: (BuildContext context, AsyncSnapshot<dynamic> snapshot) {
-                return Text(snapshot.data.toString());
-              },
-            ),
-          ],
-        ),
+        // Row(
```

Comment:
Why not comment on multiple lines together?

Reply:
I commented the future line first and then realized that I need to comment out the whole Row. I don't think this matters a lot because I will update the example in a separate ticket later anyway.